src/pages/test-report/test-report.ts

```diff
@@ -84,14 +84,22 @@ export class TestReportPage extends BasePageComponent {
     };
 
     const { candidateUntitledName$, isSeriousMode$, isDangerousMode$ } = this.pageState;
-
     const merged$ = merge(
       candidateUntitledName$,
       isSeriousMode$.pipe(map(result => this.isSeriousMode = result)),
       isDangerousMode$.pipe(map(result => this.isDangerousMode = result)),
     );
-
     this.subscription = merged$.subscribe();
+
+    this.manoeuvres$ = this.store$.pipe(
+      select(getTests),
+      select(getCurrentTest),
+      select(getTestData),
+      select('manoeuvres'),
+    );
+    this.manoeuvresSubscription = this.manoeuvres$.subscribe((result) => {
```

Comment:
Again, this would probably be better in page state + combined with a selector.

Reply:
moved to page state and added selector